ffi/wgpu.h

```diff
@@ -1,6 +1,10 @@
 #define WGPU_LOCAL
 
+<<<<<<< HEAD
```

Comment:
someone forgot to resolve the merge conflicts!

Reply:
Oh no you are seeing me struggling with the rebasing again! I am fixing it right now (hopefully)